app/services/gias_csv_importer.rb

```diff
@@ -37,10 +37,13 @@ def call
       end
 
     if invalid_records.any?
-      logger.info "Invalid rows - #{invalid_records.inspect}"
+      Rails.logger.info "Invalid rows - #{invalid_records.inspect}"
     end
+    turn_off_ar_logger
     GiasSchool.upsert_all(records, unique_by: :urn)
-    logger.info "Done!"
+    turn_on_ar_logger
```

Comment:
I think you might be able to use the `logger.silence` method here instead.

```ruby
Rails.logger.silence do
  GiasSchool.upsert_all(records, unique_by: :urn)
end
```

Reply:
Good shout